server/poll/poll.go

```diff
@@ -126,6 +126,14 @@ func (p *Poll) UpdateVote(userID string, index int) error {
 	return nil
 }
 
+// getAnswerOptionName returns answer option name (with voter count if progress setting is available)
+func (p *Poll) getAnswerOptionName(o *AnswerOption) string {
+	if p.Settings.Progress {
+		return fmt.Sprintf("%s (%d)", o.Answer, len(o.Voter))
+	}
+	return o.Answer
```

Comment:
Nit:
```suggestion
	}
	
	return o.Answer
```

Reply:
Thanks, but this blank line seems to cause style error.
https://app.circleci.com/pipelines/github/matterpoll/matterpoll/479/workflows/e7167a1b-e996-4a3c-bed6-f82305912cd4/jobs/869/parallel-runs/0/steps/0-106

I'll revert this.